upgrading/v3.1-to-v3.2.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+title: Upgrading from OpenAPI 3.1 to 3.2
+parent: Upgrading Between Versions
+nav_order: 2
+---
+
+# Upgrading from OpenAPI 3.1 to 3.2
+
+OpenAPI 3.2 introduces substantial new functionality while maintaining full backward compatibility with OpenAPI 3.1. This guide covers the upgrade process and provides a roadmap for gradually adopting new features.
+
+## Getting Started
+
+Begin by updating the version number in your OpenAPI document:
+
+```yaml
+# Before
+openapi: 3.1.1
+
+# After
+openapi: 3.2.0
+```
+
+**Key advantage**: Unlike the 3.0 to 3.1 transition, OpenAPI 3.2 does not introduce breaking changes. All existing 3.1 documents will work without modification after updating the version number.
+
+## What's New in OpenAPI 3.2
```

Comment:
I think this doc should also mention the change to the discriminator feature that allows the discriminating property to be optional, and a "defaultMapping" to specify the schema for instances that do not include a discriminating property.